### PR TITLE
add ped dynamic shadows functions

### PR DESCRIPTION
**Pull request by Proxy-99**
`setDynamicPedShadowsEnabled(bool)`
`isDynamicPedShadowsEnabled()`
`resetDynamicPedShadows()`
#2777

### CODE REVIEW

**Review by TracerDS — APPROVED** [review]

**Comment by FileEX** [comment]
What do you think about calling ``ResetDynamicPedShadows`` from ``ResetWorldProperties``?

**Comment by Proxy-99** [comment]
> What do you think about calling `ResetDynamicPedShadows` from `ResetWorldProperties`?

done!

File: Client/mods/deathmatch/logic/luadefs/CLuaWorldDefs.cpp [hunk 1]
```diff
@@ -2278,3 +2281,21 @@ void CLuaWorldDefs::ResetWorldProperties(std::optional<bool> resetSpecialWorldPr
...
+{
+    return g_pGame->GetSettings()->IsDynamicPedShadowsEnabled();
+}
+
+bool CLuaWorldDefs::ResetDynamicPedShadows() noexcept
+{
+    return g_pGame->GetSettings()->ResetDynamicPedShadows();
+}
+
+
```
Comment (FileEX) [inline Client/mods/deathmatch/logic/luadefs/CLuaWorldDefs.cpp]: Extra space

**Review by FileEX — APPROVED** [review]

File: Client/game_sa/CSettingsSA.cpp [hunk 2]
```diff
@@ -316,6 +316,20 @@ void CSettingsSA::SetDynamicPedShadowsEnabled(bool bEnable)
     m_bDynamicPedShadowsEnabled = bEnable;
 }
 
+bool CSettingsSA::GetDynamicPedShadowsEnabledByVideoSetting() const noexcept
```
Comment (TheNormalnij) [inline Client/game_sa/CSettingsSA.cpp]: `Get` -> `Is`

**Review by TheNormalnij — APPROVED** [review]

**Comment by Fernando-A-Rocha** [comment]
hello, I noticed wiki pages missing @Proxy-99 
https://wiki.multitheftauto.com/wiki/SetDynamicPedShadowsEnabled
https://wiki.multitheftauto.com/wiki/IsDynamicPedShadowsEnabled
https://wiki.multitheftauto.com/wiki/ResetDynamicPedShadows

**Pull request merged**